share/ci/install/pypicongpu.sh

```diff
@@ -25,14 +25,17 @@ export PIC_EXAMPLES=$PICSRC/share/picongpu/examples
 cd $CI_PROJECT_DIR
 
 # use miniconda as python environment
-apt update && apt install -y wget
+apt update && apt install -y curl
 cd /tmp/
-wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
-chmod u+x Miniconda3-latest-Linux-x86_64.sh
-./Miniconda3-latest-Linux-x86_64.sh -b -p /miniconda3
-export PATH=/miniconda3/bin:$PATH
+curl -Ls https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba
+export MAMBA_ROOT_PREFIX=/tmp/mamba-forge/
+mkdir -p "${MAMBA_ROOT_PREFIX}"
+eval "$(./bin/micromamba shell hook -s posix)"
+export PATH=$(pwd -P)/bin:$PATH
+alias conda="micromamba"
```

Comment:
Why do you use an alias?
In the other PR, the `conda` is replaced by `micromamba`: https://github.com/ComputationalRadiationPhysics/picongpu/pull/5041/files
Also, I think the alias is confusing.

Reply:
It's supposed to be a minimally invasive hot fix. If this passes the CI, we can make a `sed -i` but for now I wanted to have the least possible ways for this to fail.